### PR TITLE
Local cache

### PR DESCRIPTION
**Pull request by anwzhang**

### CODE REVIEW

File: sdk/PrebidMobile/PBBidManager.m [hunk 1]
```diff
@@ -344,19 +339,9 @@ - (void)clearBidOnAdObject:(NSObject *)adObject {
...
-                if ([keyword hasPrefix:reservedKey]) {
-                    [mutableKeywords removeObject:keyword];
-                    return;
-                }
-            }
-        }];
         SEL setKeywords = NSSelectorFromString(@"setKeywords:");
         if ([adObject respondsToSelector:setKeywords]) {
-            [adObject performSelector:setKeywords withObject:[mutableKeywords componentsJoinedByString:@","]];
+            [adObject performSelector:setKeywords withObject:nil];
```
[the remark this answers is not in the document]
Comment (ppuviarasu) [reply sdk/PrebidMobile/PBBidManager.m]: Why shouldn't this be cleared? I want to clear all the keywords that we haev set & replace with new ones. How come this is wrong?

[the remark this answers is not in the document]
Comment (ppuviarasu) [reply sdk/PrebidMobile/PBBidManager.m]: Previous logic was clearing only "hb_bidder"; "hb_cache_id";"hb_pb";

But not everything
Its ideal to clear everything & add fresh...

[the remark this answers is not in the document]
Comment (anwzhang) [reply sdk/PrebidMobile/PBBidManager.m]: Does this clear client's keywords too?

[the remark this answers is not in the document]
Comment (ppuviarasu) [reply sdk/PrebidMobile/PBBidManager.m]: yes this logic will... I have modified the logic...

**Pull request merged**